ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -244,24 +246,18 @@ def install_internal_cluster(self):
 
         """
         logger.info("Click Internal")
-        if (
-            self.operator_name == ODF_OPERATOR
-            and self.ocp_version_semantic != version.VERSION_4_11
-        ):
+        if self.operator_name == ODF_OPERATOR:
             self.do_click(
                 locator=self.dep_loc["internal_mode_odf"], enable_screenshot=True
             )
-        elif self.ocp_version_semantic != version.VERSION_4_11:
+        else:
             self.do_click(locator=self.dep_loc["internal_mode"], enable_screenshot=True)
 
-        if self.ocp_version_semantic != version.VERSION_4_11:
-            logger.info("Configure Storage Class (thin on vmware, gp2 on aws)")
-            self.do_click(
-                locator=self.dep_loc["storage_class_dropdown"], enable_screenshot=True
-            )
-            self.do_click(
-                locator=self.dep_loc[self.storage_class], enable_screenshot=True
-            )
+        logger.info("Configure Storage Class (thin on vmware, gp2 on aws)")
+        self.do_click(
+            locator=self.dep_loc["storage_class_dropdown"], enable_screenshot=True
```

Comment:
Is the screenshot for debugging purpose? If yes please remove it

Reply:
Incase the test fails?

Reply:
No, for example we want to verify we have gp3 on AWS... so we can see all the options.. the default sc  in our deployment is gp2